.happy/terraform/envs/dev/main.tf

```diff
@@ -13,7 +13,7 @@ module stack {
   frontend_url                 = "https://cellxgene.dev.single-cell.czi.technology"
   backend_url                  = "https://api.cellxgene.dev.single-cell.czi.technology"
   stack_prefix                 = ""
-  batch_container_memory_limit = 60000
+  batch_container_memory_limit = 120000
```

Comment:
Usually prod resources need to be larger than dev!  It feels odd having to give in to dev's messy data state in this way.  Maybe comment why this is larger than prod? 😬

Reply:
Actually, prod has 300G!

Reply:
Ha, ok! I feel a little better about this then. 😆